bats/fb-katello-content.bats

```diff
@@ -116,6 +116,54 @@ setup() {
     --content-view="${CONTENT_VIEW}" --to-lifecycle-environment="${LIFECYCLE_ENVIRONMENT}" --from-lifecycle-environment="Library"
 }
 
+@test "export content view version" {
+  hammer content-export complete version --organization="${ORGANIZATION}" \
+    --content-view="${CONTENT_VIEW}" --version="1.0" | grep -q "Generated"
+}
+
+@test "create skeleton org, product, content view, and repo for import" {
+  hammer organization create --name="${IMPORT_ORG}" | grep -q "Organization created"
+  hammer product create --organization="${IMPORT_ORG}" --name="${PRODUCT}" | grep -q "Product created"
+  hammer content-view create --import-only --organization="${IMPORT_ORG}" \
+    --name="${CONTENT_VIEW}" | grep -q "Content view created"
+  hammer repository create --organization="${IMPORT_ORG}" \
+    --product="${PRODUCT}" --content-type="yum" --name "${YUM_REPOSITORY}" \
+    --url https://jlsherrill.fedorapeople.org/fake-repos/needed-errata/ | grep -q "Repository created"
+  repo_id=$(hammer --csv --no-headers repository list --organization="${IMPORT_ORG}" \
+  | grep ${YUM_REPOSITORY} | cut -d, -f1)
+  hammer content-view add-repository --organization="${IMPORT_ORG}" \
+  --name="${CONTENT_VIEW}" --repository-id=$repo_id | grep -q "The repository has been associated"
+}
+
+@test "import the exported content view" {
+  latest_export=$(hammer --output csv --no-headers content-export list --content-view "${CONTENT_VIEW}" --organization "${ORGANIZATION}"\
+   --content-view-version="1.0" --per-page=1 --order="id DESC")
```

Comment:
```suggestion
   --content-view-version="1.0" --fields=id,path --per-page=1 --order="id DESC")
```
specify the fields so that we are not stuck by the format of the output